README.md

```diff
@@ -8,7 +8,7 @@ operating system settings. The plugin distinguishes between `Light`, `Dark` and
 the theme used for each mode can be customized.
 This plugin currently works for Windows and macOS.
 
-Linux support is both limited and experimental. At the moment, the only Linux desktop environment supported by this plugin is Gnome.
+Linux support is both limited and experimental. At the moment, Linux desktop environments based on Gtk and has an [XSettings](https://www.freedesktop.org/wiki/Specifications/xsettings-spec/) daemon are supported.
```

Comment:
This sounds odd. How about "Linux desktop environments based on Gtk, which have an [XSettings](https://www.freedesktop.org/wiki/Specifications/xsettings-spec/) daemon are supported."

Reply:
I wonder if I should expand on that:

1. An XSettings daemon (e.g. `gsd-xsettings` for Gnome; `xfsettingsd` for Xfce; or even standalone ones like https://github.com/derat/xsettingsd) is required because it is the one that actually sends out the notifications when a theme (amont other settings) changes;
2. Conversely, you can run a desktop environment without XSettings daemon (e.g. to my knowledge LXDE does not have one by default), in which case theme changes are not "real time" (i.e. they usually  takes effect only for new apps launched, since running apps do not get notified)
3. I haven't verified it, but I think even non-Gtk native environments like KDE _should_ work with this, _if_ it runs such a daemon (e.g. https://github.com/KDE/xsettings-kde)

so really the dominating factor is not Gtk (we just use it to get notified), but rather the XSettings daemon (who actually sends out the notifications)

Reply:
Thank you for the clarification. Nonetheless the setence as it is isn't grammatically correct. I also think it would be worth while to elaborate a bit on what is needed to use the plugin. This could also be added to https://github.com/weisJ/auto-dark-mode/blob/master/plugin/src/main/resources/META-INF/plugin.xml